{"name": "test_basic_authentication[/endpoint2]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 404 == 200\n +  where 404 = <Response [404]>.status_code", "trace": "endpoint = '/endpoint2', auth = <requests.auth.HTTPBasicAuth object at 0x0000028310842A10>\n\n>   ???\nE   assert 404 == 200\nE    +  where 404 = <Response [404]>.status_code\n\nD:\\PythonTutorial\\PythonApiAutomation\\restApiRequests\\test_basic_auth.py:20: AssertionError"}, "parameters": [{"name": "endpoint", "value": "'/endpoint2'"}], "start": 1693915199029, "stop": 1693915200442, "uuid": "14af3ca3-95c4-48da-af9c-326b5e97a339", "historyId": "5ec41132431af4552557b17f99f49f29", "testCaseId": "929201250b532c178ccd26aaec38b978", "fullName": "tests.restApiRequests.test_basic_auth#test_basic_authentication", "labels": [{"name": "parentSuite", "value": "tests.restApiRequests"}, {"name": "suite", "value": "test_basic_auth"}, {"name": "host", "value": "IB-Pune-LAP-163"}, {"name": "thread", "value": "22040-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.restApiRequests.test_basic_auth"}]}